lib/tasks/data_hygiene/publishing_delay_reporter.rb

```diff
@@ -0,0 +1,32 @@
+class Tasks::DataHygiene::PublishingDelayReporter
+  def report
+    now = Time.now
```

Comment:
We should use `Time.zone.now` to make sure we take timezones into account (and I think Rails will do utc-ification on them when passing those into db queries).

Reply:
I might well be missing something (timezones are confusing), but won't you get the same result whether you use `Time.now` and `Time.zone.now` precisely because of Rails utc-ification?

If I understand it correctly, `Time.now` will get the current time in the system's timezone, and `Time.zone.now` will get the current time in Rails's timezone, but they both refer to the same instant in time. So it shouldn't matter which one you put into the DB query?

I've played about with timezones in a Rails console and couldn't find a query which gave different results.

Reply:
Maybe if our app and servers are all set to utc then it doesn't actually matter because the timezones are always UTC so no conversion actually happens.  It might be more of an issue in summertime when UK local time isn't UTC.  I agree it'd be minor though, as I don't think we'd miss out on any data.

I might be cargo-culting this without much internal debate just because I've had it drilled into me by loads of blog posts and presentations that in Rails we should always use `Time.zone` and never `Time`.  Loathe as I am to point to rubocop as an arbiter of "good", it does have a cop for that says the same: https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/rails/time_zone.rb, which we'd eventually fall foul of when/if we upgrade the version of govuk-lint/rubocop on this project.

Reply:
OK, I've updated it. 👍 I don't think it'll affect this query, but I agree that it's better to use in general (certainly for anything user-facing).